src/app/page.tsx

```diff
@@ -1,14 +1,35 @@
 "use client";
 
 import React from 'react';
-import { Box } from '@mui/material';
-
+import BasicTable from '@/components/molecules/my-Table/table';
+import MyPageTitle from '@/components/molecules/title/my-page-title';
+import { Section } from '@/components/molecules/section/section';
+import { Typography, Box } from '@mui/material';
 
 
 export default function Home() {
   return (
-    <Box>
-      
-    </Box>
+    <>
+      <Box component="div" sx={{mb: "40px"}}>
+        <MyPageTitle title="TP3-Intégration de l'API" />
+      </Box>
+      <Section title="Description du projet">
```

Comment:
section ? is this a MUI component or bootstrap ? Is there any difference between the container or like the Box and the section ?

Reply:
C'est MUI, cette ligne de code est propre a MUI : **component="div" sx={{mb: "40px"}}**. Une box c'est pour avoir un container flexible et fait partie de MUI. La section est pour regrouper des éléments liés entre eux, tels que des en-têtes, des sous-titres, des listes ou des blocs de contenu. Il ne fait pas partie de MUI.

Reply:
My bad ! Thanx